### PR TITLE
12 add es lint

### PR DESCRIPTION
**Pull request by zrealberg**
I installed ESLint and configured it to use HRs rules. I think this is only set up to work by running the command `eslint client/components/App.jsx`, which is to say it won't run automatically upon commit. Do we want that?

### CODE REVIEW

File: client/components/App.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ class App extends React.Component {
       </div>
     )
   }
-  
+
```
Comment (cchauche) [inline client/components/App.jsx:15]: Not going to split hairs but if you wanted some practice fixing a pull request I don't think any changes to App.jsx need to be a part of this pull request.

**Review by cchauche — APPROVED** [review]

File: client/components/App.jsx (hunk 1 above)
Comment (zrealberg) [reply client/components/App.jsx:15]: @cchauche Let me try to fix this. I certainly didn't mean to include this change in my update.

**Comment by zrealberg** [comment]
I added an eslintignore file, and I think I undid whatever change I made to App.jsx, so it should be good now?

**Review by rhunce — APPROVED** [review]
Looks good to me!

**Pull request merged**